front/lib/api/assistant/configuration.ts

```diff
@@ -1412,6 +1411,25 @@ function renderRetrievalTimeframeType(
   return timeframe;
 }
 
+export async function getAgentsFirstCreationDate(agentIds: string[]) {
```

Comment:
Seen IRL, can we avoid introducing the support to sort on `createdAt` for now? This endpoint is already pretty slow and this query does not use any index.